specs/deneb/beacon-chain.md

```diff
@@ -358,7 +333,7 @@ def process_execution_payload(state: BeaconState, body: BeaconBlockBody, executi
 
     # Verify the execution payload is valid
     # [Modified in Deneb:EIP4844] Pass `versioned_hashes` to Execution Engine
-    # [Modified in Deneb:EIP4788] Pass `parent_beacon_block_root` to Execution Engine
+    # [Modified in Deneb:EIP4788] Verify `parent_beacon_block_root` with executon payload's
```

Comment:
Can remove this comment when you hoist the verification into `process_execution_payload` above